v5_ws_private.go

```diff
@@ -223,10 +227,42 @@ func (s *V5WebsocketPrivateService) Run() error {
 			return err
 		}
 	}
-
 	return nil
 }
 
+func (s *V5WebsocketPrivateService) waitForSubConfirmation(topic string) error {
+	// Read 30 times to wait for the subscription confirmation.
```

Comment:
Why 30 reads? Could we instead have a default time on how long we are willing to wait for subscribe message to come instead?